### ISSUE TITLE
Remove link to Liberty App Accelerator

### ISSUE THREAD

**Issue by cherylking** [OpenLiberty/guide-gradle-intro#29]
Near the bottom of this guide, there is a link to the Liberty App Accelerator. That link should be removed, as the tool no longer exists.

**Comment by justineechen** [comment]
The link has been removed on the multipane version of the guides and will be removed once the multipane branch is merged into master (in the coming weeks).  Thank you for letting us know!

**Comment by justineechen** [comment]
The link is removed on the published site: https://openliberty.io/guides/gradle-intro.html
